sdk/communication/azure-communication-common/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## 1.0.0-beta.4 (Unreleased)
 ### Breaking Changes
 - Renamed `CommunicationUserCredential` to `CommunicationTokenCredential`
+- Replaced constructor `CommunicationTokenCredential(TokenRefresher tokenRefresher, String initialToken,	boolean refreshProactively)` with `CommunicationTokenCredential(CommunicationTokenRefreshOptions tokenRefreshOptions, String initialToken)`
```

Comment:
One more change needed after the last commit.

```suggestion
- Replaced constructor `CommunicationTokenCredential(TokenRefresher tokenRefresher, String initialToken,	boolean refreshProactively)` with `CommunicationTokenCredential(CommunicationTokenRefreshOptions tokenRefreshOptions)`
```

We can also combine both replacements into one line.

Reply:
Good catch! Thank you :)